src/excalidraw-app/collab/Portal.tsx

```diff
@@ -63,6 +64,7 @@ class Portal {
       return;
     }
     this.queueFileUpload.flush();
+    this.socket.off();
```

Comment:
probably unnecessary as one would hope `socket.close()` does this for us, but one never knows.